components/mediation-ui/org.wso2.carbon.inbound.ui/src/main/java/org/wso2/carbon/inbound/ui/internal/InboundManagementClient.java

```diff
@@ -361,4 +366,43 @@ public boolean updteInboundEndpoint(String name, String sequence, String onError
         }
         return inboundNameList;
     }
-}
+
+    public String getKAFKASpecialParameters() {
+        String specialParamsList = "";
+        loadProperties();
+        if (prop != null) {
+            String strKey = "kafka.mandatory";
+            String strLength = prop.getProperty(strKey);
+            Integer iLength = null;
+            if (strLength != null) {
+                try {
+                    iLength = Integer.parseInt(strLength);
+                } catch (Exception e) {
```

Comment:
I used NumberFormateException at this point in another PR. https://github.com/wso2/carbon-mediation/pull/460